### DISCUSSION TITLE
Welcome to macOS_Big_Sur_icons_replacements Discussions!

### DISCUSSION THREAD

**Discussion by elrumo** [elrumo/macOS_Big_Sur_icons_replacements#978] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by joshua-whitney** [comment]
Any plans to make a fork for Linux??? That would be super cool!

**Reply by elrumo** [reply]
Yep, that's exactly what I have planned :)

**Comment by imabhaysutar** [comment]
Hey, it would be fun to fork it as MacOS X icons for legacy icons. I see a lot of apps switching to Big Sur style icons, but that does not look well on Catalina and below. Keep up the great work 🔥

**Reply by elrumo** [reply]
That's a good idea! I can add a subdomain so it's something like legacy.macOSicon.com. I probably won't get the time to do it until next month, but I like the idea!

**Reply by imabhaysutar** [reply]
that sounds great, would love to see that. 💯 Thought of this cause I wanted to upload a old icon but thought that's kinda unorganised me. : )
